corehq/apps/reports/util.py

```diff
@@ -125,6 +128,18 @@ def possible_usernames():
         return HQUserType.human_readable[HQUserType.ADMIN]
 
 
+def get_user_id(form_id):
```

Comment:
nit: suggested rename to `get_user_id_from_form`

Reply:
https://github.com/dimagi/commcare-hq/pull/30965/commits/a3df1bf86e1625e3e33d5a31a9f185e5b4208f08